composer.json

```diff
@@ -49,7 +49,8 @@
 	"autoload": {
 		"psr-4": {
 			"Friendica\\": "src/",
-			"Friendica\\Test\\": "tests/"
+			"Friendica\\Test\\": "tests/",
+			"Friendica\\Addon\\": "addon/"
```

Comment:
Oooh, I like where this is going. But shouldn't you add this in `friendica-addons` instead?

Reply:
mmh.. the composer.json  file is in core repo.. this is needed to autoload classes from addons.
The idea is to allow addons to register a storage backend, which will be autoloaded from addon folder, under namespace `Friendica\Addon\addonname`, e.g. `Friendica\Addon\S3Storage\S3StorageBackend.php`

Reply:
> It's a little late but have considered https://github.com/thephpleague/glide ?

It's a little late to reply, but no, I never heard about. I didn't read all about it, but looks interesting...